src/containers/pages/InterventionPlan/UpdatePlan/tests/index.test.tsx

```diff
@@ -927,4 +927,59 @@ describe('components/InterventionPlan/UpdatePlan', () => {
     // check submit button - should not be disabled
     expect(wrapper.find('#planform-submit-button button').prop('disabled')).toEqual(false);
   });
+
+  it('should pass correct array to hide fields for case triggered draft plans', async () => {
```

Comment:
let's have additional tests here showing those fields are hidden on the form as well.

Reply:
Added tests for some fields, to checks if they are hidden or not.